banners/mobile/event_map.ts

```diff
@@ -20,9 +21,9 @@ export default new Map<string, TrackingEventConverterFactory>( [
 	[ BannerSubmitEvent.EVENT_NAME, ( e: BannerSubmitEvent ): WMDESizeIssueEvent => {
 		switch ( e.feature ) {
 			case 'UpgradeToYearlyForm':
-				return new WMDESizeIssueEvent( `submit-${e.userChoice}`, null, 1 );
+				return new WMDESizeIssueEvent( `submit-${e.userChoice}`, createViewportInfo(), 1 );
```

Comment:
hope these changes here are sufficient for the "submit button logs the viewport dimensions"